README.md

```diff
@@ -5,11 +5,31 @@
 2. AccountModifiers.sol - a simple contract to hold fee/rebate modifiers
 3. TokenTemplate.sol - a sample ERC20 token contract to be used in testing only
 
+## Install Contracts
+
 ## Testing
-1. Install [truffle](http://truffleframework.com/) 3.4.6
-2. Install [testrpc](https://github.com/ethereumjs/testrpc) 3.9.2
-2. Launch testrpc, unlocking first several accounts with `testrpc --secure -u 0 -u 1 -u 2 -u 3 -u 4`
-3. Launch `truffle test` from command line
+1. `$ npm install -g truffle@3.4.6`
+2. `$ npm install -g ganache-cli`
+3. `$ npm install`
+4. Launch ganache-cli, unlocking first several accounts with `ganache-cli --secure -u 0 -u 1 -u 2 -u 3 -u 4`
+5. Launch `truffle test` from command line
+6. Look for this and copy it:
+```
+////////////////////////
+////////////////////////
+////////////////////////
+TOKEN_ADDRESS
+0xXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+////////////////////////
+////////////////////////
+////////////////////////
+```
+7. Put this address in `tokenstore/api/src/lib/config.js` so that this line is updated:
```

Comment:
Contract is a public repo supposed to be able to being used by other users without any connection to our other repos (private at the moment). This should be in api repo readme.

Reply:
It seems for my current setup these READMEs are coupled. I may be bringing in the `/privatenet` contract in incorrectly. Currently this is how I know to update the `tokenstore/api/src/lib/config.js` file. Either this README references `tokenstore/api` or `tokenstore/api` references this one